openlibrary/plugins/openlibrary/js/ol.js

```diff
@@ -390,186 +390,251 @@ function passwordHide(){
 
 $().ready(function(){
     var cover_url = function(id) {
-	return '//covers.openlibrary.org/b/id/' + id + '-S.jpg'
+        return '//covers.openlibrary.org/b/id/' + id + '-S.jpg'
     };
 
     var capitalize = function(word) {
-	return word.charAt(0).toUpperCase() + word.slice(1);
+        return word.charAt(0).toUpperCase() + word.slice(1);
+    }
+
+    // Maps search facet label with value
+    var searchFacets = {
+        'title': 'books',
+        'author': 'authors',
+        'subject': 'subjects',
+        'advanced': 'advancedsearch'
+    };
+
+    var composeSearchUrl = function(q, json, limit) {
+        var facet_value = searchFacets[localStorage.getItem("facet")];
+        var url = ((facet_value === 'books')? '/search' : "/search/" + facet_value);
```

Comment:
Don't need surrounding parentheses. Add a space before `?`. Just use single-quotes for the string.